worker/src/RTC/SvcConsumer.cpp

```diff
@@ -581,7 +581,10 @@ namespace RTC
 		auto previousSpatialLayer  = this->encodingContext->GetCurrentSpatialLayer();
 		auto previousTemporalLayer = this->encodingContext->GetCurrentTemporalLayer();
 
-		if (!packet->ProcessPayload(this->encodingContext.get()))
+		bool marker{ false };
```

Comment:
Same here.

Reply:
With VP9 the marker may be set in `packet->ProcessPayload(...)`

https://github.com/versatica/mediasoup/blob/v3/worker/src/RTC/Codecs/VP9.cpp#L353